### ISSUE TITLE
Feature request: Flatten nested json (sounds and observation_sounds)

### ISSUE THREAD

**Issue by arky** [pyinat/pyinaturalist-convert#162]
### Feature description

Please provide features to flatten deeply nested json structures of `sounds`  and `observation_sounds`

### Use case
_Is there a specific goal that this would help you accomplish, or can you provide any other context about how you would like to use this feature?_
Testcase

```python
from pyinaturalist import get_observations
from pyinaturalist_convert import *

observations = get_observations(user_id='arkyar', sounds=True, pages=1)
df = to_dataframe(response)

df.filter(like='sound')

```

### Workarounds
_Is there an existing workaround to accomplish this?_

You'll need to use third party libraries as `pd.json_normalize` might not work for this?

**Comment by JWCook** [comment]
Sure, I'm already doing this for photos, so it should be easy enough to do for sounds.

Currently, flattened photos are in the format `{'photo_id': 'photo_url'}`, so doing the same for sounds would look like:
```py
{
  'sounds': [
    {12345: 'https://static.inaturalist.org/sounds/sound1.wav'},
    {67890: 'https://static.inaturalist.org/sounds/sound2.wav'},
    ...,
  ]
}
```

Does that work for you?

**Comment by arky** [comment]
@JWCook That works for me.

**Comment by JWCook** [comment]
Sorry for the long delay! This is now available in v0.6.3.

**Comment by arky** [comment]
Sweet @JWCook You are awesome. Thank you.
